doc/dev/quick_start.rst

```diff
@@ -249,15 +249,15 @@ are supported in Catalyst, although not all features are supported for all measu
    :header-rows: 0
 
    * - :func:`qml.expval() <pennylane.expval>`
-     - The expectation value of all observables is supported.
+     - The expectation value of all observables and finite-shots are supported.
```

Comment:
Is this more accurate?
```suggestion
     - The expectation value of all observables is supported analytically, as well as with finite-shots except for Hermitian.
```

Reply:
This is not true for all backend devices though. I added the details for each device to `runtime/README`.

Reply:
I've updated the suggestion, how does that sound? The current formulation doesn't read that well.

Reply:
Sounds good! I'll update it in other places too. Thanks :+1: